### ISSUE TITLE
Use rem for font size & related rules units

### ISSUE THREAD

**Issue by WilliamKelley** [prenda-school/prenda-spark#12]
Asana Card: https://app.asana.com/0/1200012653024704/1200065794114872

Accessibility is negatively impacted when `font-size` and related css rules like `line-height` are set with `px` units. See [MDN's warning](https://developer.mozilla.org/en-US/docs/Web/CSS/font-size#pixels).

Values that I see exist in the codebase right now seem to agree on `16px` as the standard font size. Matching up with common browser defaults, that's equivalent to `1rem`. Users commonly change their browser default for accessibility reasons and `rem` will adjust according. [Here's an aid](http://www.standardista.com/px-to-rem-conversion-if-root-font-size-is-16px/) for ratio conversions.

I don't think there's a need to worry if `16px` should be our standard or not. I'll figure that much out when I get the font files hooked up and set with our theme.

**Comment by WilliamKelley** [comment]
This has been solved over the course of many PRs
